osu.Game/Screens/Play/HUD/ArgonSongProgressBar.cs

```diff
@@ -74,6 +85,7 @@ public ArgonSongProgressBar(float barHeight)
         private void load(OsuColour colours)
         {
             catchUpColour = colours.BlueDark;
+            inputManager = GetContainingInputManager();
```

Comment:
There's no way this is required to make this work. You can get the local position of the mouse by using `OnMouseMove` instead and handling the event.